src/clientprocess/from_static_lint.jl

```diff
@@ -162,3 +162,11 @@ function save_store_to_disc(store, file)
     serialize(io, store)
     close(io)
 end
+
+function pkg_name(pkg)
+    return first(pkg)
+end
+
+function pkg_uuid(pkg)
+    return string(last(pkg))
+end
```

Comment:
I think it is just the other way around, i.e. isn't the first element in the `Pair` the uuid and the last one the name?